editor/src/components/canvas/controls/comment-popup.tsx

```diff
@@ -0,0 +1,99 @@
+import '@liveblocks/react-comments/styles.css'
+import React from 'react'
+import { Substores, useEditorState } from '../../editor/store/store-hook'
+import { CanvasOffsetWrapper } from './canvas-offset-wrapper'
+import { isCommentMode } from '../../editor/editor-modes'
+import { ClientSideSuspense } from '@liveblocks/react'
+import { useCreateThread, useThreads } from '../../../../liveblocks.config'
+import type { ComposerSubmitComment } from '@liveblocks/react-comments'
+import { Comment, Composer } from '@liveblocks/react-comments'
+import { stopPropagation } from '../../inspector/common/inspector-utils'
+import { UtopiaTheme } from '../../../uuiui'
+import { ErrorBoundary } from '../../../utils/react-error-boundary'
+import { useCanvasCommentThread } from '../../../core/commenting/comment-hooks'
+import { isLoggedIn } from '../../editor/action-types'
+
+export const CommentPopup = React.memo(() => {
+  const mode = useEditorState(
+    Substores.restOfEditor,
+    (store) => store.editor.mode,
+    'CommentIndicator mode',
+  )
+
+  if (!isCommentMode(mode) || mode.location == null) {
+    return null
+  }
+  const { location } = mode
+  if (location == null) {
+    return null
+  }
+
+  return (
+    <CanvasOffsetWrapper>
+      <div
+        style={{
+          position: 'absolute',
+          top: location.y,
+          left: location.x + 30,
+          cursor: 'text',
+          minWidth: 250,
+          boxShadow: UtopiaTheme.panelStyles.shadows.medium,
+        }}
+        onKeyDown={stopPropagation}
+        onKeyUp={stopPropagation}
+        onMouseUp={stopPropagation}
+      >
+        <ErrorBoundary fallback={<div>Can not load comments</div>}>
+          <ClientSideSuspense fallback={<div>Loading…</div>}>
+            {() => <CommentThread x={location.x} y={location.y} />}
+          </ClientSideSuspense>
+        </ErrorBoundary>
+      </div>
+    </CanvasOffsetWrapper>
+  )
+})
+
+interface CommentThreadProps {
+  x: number
+  y: number
+}
+
+function CommentThread({ x, y }: CommentThreadProps) {
+  const thread = useCanvasCommentThread(x, y)
+  const createThread = useCreateThread()
+
+  // TODO: Unify getting name in different multiplayer components
+  const loginState = useEditorState(
+    Substores.userState,
+    (store) => store.userState.loginState,
+    'CommentThread loginState',
+  )
+
+  const name = isLoggedIn(loginState) ? loginState.user.name : null
```

Comment:
this can be obtained from the presence on the `name` key (and with https://github.com/concrete-utopia/utopia/pull/4495 there's a helper to normalize it)